lib/OpenQA/WebAPI.pm

```diff
@@ -501,6 +501,8 @@ sub startup ($self) {
       ->name('apiv1_put_parent_group_comment')->to('comment#update');
     $api_ra->delete('/parent_groups/<parent_group_id:num>/comments/<comment_id:num>')
       ->name('apiv1_delete_parent_group_comment')->to('comment#delete');
+    $api_ru->post('/comments')->name('apiv1_post_comments')->to('comment#create_many');
```

Comment:
`/jobs/comments` to keep it consistent with what it does.

Reply:
I wouldn't do that because in the future we might want to allow specifying group IDs as well. Then it makes most sense to still have only one route for posting comments in general.

The path `/jobs/comments` is also in conflict with `/jobs/<job_id>/…`. And yes, this is doable. However, I find it not ideal if "comments" and a job ID can occur at the same path element.

Reply:
fair enough